### ISSUE TITLE
Trying to get a token 

### ISSUE THREAD

**Issue by KurtC0ba1n** [atom/teletype#366]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Followed all applicable steps in the [debugging guide][debugging-guide]
    * Checked the FAQs on the [message board][message-board] for common solutions
    * Checked that your issue [isn't already filed][already-filed]?

[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/teletype/issues

### Description

trying to get a token using Github and teletype

### Steps to Reproduce

1. click on the antenna icon
2. click on the link to get a token 
3. open my browser and ask me to login with my github account 

**Expected behavior:**

get a token 

**Actual behavior:**

huuuu... get a 404 not found at this page https://teletype.atom.io/login/oauth/authorize

**Reproduces how often:**

I tried like 5/6 times

### Versions

I don't think this come from Atom but more from your website

**Comment by jasonrudolph** [comment]
Thanks for letting us know about this issue. I'll try to help. Can you give this a try:

1. Log out of github.com (https://github.com/logout)
2. Then, go to https://teletype.atom.io/login and follow the prompts

Does that resolve the issue for you?

**Comment by KurtC0ba1n** [comment]
not at all, I logged out and tried to login 3 times with my pseudo and my email, checked 4 time the password and I still get a 404 not found

**Comment by jasonrudolph** [comment]
@KurtC0ba1n: Would you mind sharing a gif or video of the behavior you're seeing? I'm hoping that it might help us better understand the exact flow so that we can diagnose the issue.

**Comment by coinGuyBri** [comment]
I also have a similar issue. On accident, I used someone else's token during setup. How do I overwrite or clear the token? Thank you

**Comment by rsese** [comment]
@coinGuyBri: From the command palette, run the `Teleteype: Sign out` command.
